src/viz-parole-population/VizParolePopulation.js

```diff
@@ -141,23 +141,23 @@ ParoleDemographicBarChart.defaultProps = {
 };
 
 export default function VizParolePopulation({
-  data: { populationDemographics, districtOffices },
-  districtId,
-  onDistrictClick,
+  data: { populationDemographics, paroleOffices },
+  officeId,
+  onOfficeClick,
 }) {
-  const districtTotals = populationDemographics
+  const officeTotals = populationDemographics
     .filter(recordIsTotal)
     .map((record) => {
-      const districtData = districtOffices.find(
+      const officeData = paroleOffices.find(
         // these are stored as both strings and numbers;
         // doing an extra typecast here just to be safe
         (office) => `${office.district}` === `${record.district}`
       );
-      if (districtData) {
+      if (officeData) {
         return {
-          district: `${record.district}`,
-          lat: districtData.lat,
-          long: districtData.long,
+          office: `${record.district}`,
```

Comment:
There are a few slightly awkward naming disconnects like this once because the actual data uses the `district` key when referring to the office's identifier